redact-wasm/index.ts

```diff
@@ -0,0 +1,69 @@
+import * as core from "./wasm/arcjet_analyze_bindings_redact.component.js";
+import type {
+  ImportObject,
+  RedactedSensitiveInfoEntity,
+  RedactSensitiveInfoConfig,
+  SensitiveInfoEntity,
+} from "./wasm/arcjet_analyze_bindings_redact.component.js";
+
+import { wasm as componentCoreWasm } from "./wasm/arcjet_analyze_bindings_redact.component.core.wasm?js";
+import { wasm as componentCore2Wasm } from "./wasm/arcjet_analyze_bindings_redact.component.core2.wasm?js";
+import { wasm as componentCore3Wasm } from "./wasm/arcjet_analyze_bindings_redact.component.core3.wasm?js";
+import { ArcjetRedactCustomRedact } from "./wasm/interfaces/arcjet-redact-custom-redact.js";
+
+// TODO: Do we actually need this wasmCache or does `import` cache correctly?
```

Comment:
```suggestion
```
We don't need this TODO. Since the implementations are split for the runtime, this obviously needs to cache because the fetch-initialization is expensive.